src/gt4py/backend/base.py

```diff
@@ -878,7 +878,7 @@ def generate_imports(self) -> str:
         source = """
 from gt4py import utils as gt_utils
         """
-        if self.implementation_ir.multi_stages:
+        if self.module_info or self.implementation_ir.multi_stages:
```

Comment:
I might be missing something, but shouldn't this same fix be applied too in the `generate_implementation()` method (line 905) ?

Reply:
Nice catch! Updated...

Reply:
Forget it

> I might be missing something, but shouldn't this same fix be applied too in the `generate_implementation()` method (line 905) ?

Forget about it. It is not needed.